cypress/utils/plugins/anomaly-detection-dashboards-plugin/commands.js

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
What about putting all of these files (`commands.js`, `constants.js`, `helpers.js`, etc.) in a `utils` dir per plugin (`cypress/plugins/anomaly-detection-dashboards-plugin/utils/*`)? 

To me, it seems these should be bundled in one location. Lmk what you think.

Reply:
If so, can we move all `commands/js` / `constants.js` / `helpers.js` files to a single directory so it's not split up?

Reply:
All move to `utils`